content/04.baselines.md

```diff
@@ -1,14 +1,16 @@
-For example, a researcher could first a build a logistic regression model using the same software framework planned for DL to first evaluate the classification performance.
-This approach will help researchers with assessing the complexity of the task at hand and debugging more complex DL architectures.
-The utility of these methods is evidenced by the recent development of hybrid models which combine DL and simpler models to improve robustness, interpretability, and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
-Depending on the amount of available data and the type of tasks, DL models may not necessarily perform the best.
-As an illustration, the simple baseline models by Rajkomar et al. [@doi:10.1038/s41746-018-0029-1] achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many.
+Since deep learning requires practitioners to consider a larger number and variety of tuning parameters compared to more traditional methods, it is easy to fall into the trap of performing an unnecessarily convoluted analysis. 
+Hence, before applying deep learning to a given problem, we highly recommend implementing a simple model at the beginning of each study to establish adequate performance baselines.
+More concretely, a researcher may wish to implement and fit a logistic regression model using the same software framework planned for DL first to evaluate the classification performance.
+Establishing such a performance baseline may help researchers with assessing the complexity of the task at hand as well as potential technical challenges surrounding data processing-related steps, which can prove useful for implementing more complex DL architectures later on.
+The utility of this approach is evidenced by the recent development of hybrid models that combine DL and simpler models to improve generalization performance, model interpretability, and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
+Depending on the amount and the nature of the available data, as well as the task to be performed, deep learning may not always be able to outperform conventional methods.
+As an illustration, Rajkomar et al. [@doi:10.1038/s41746-018-0029-1] found that simpler baseline models achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many. 
+Another example is provided by Koutsoukas et al., who benchmarked several traditional machine learning approaches against deep neural networks for modeling bioactivity data and found that, while well tuned deep learning approaches tend to outperform conventional classifiers on a moderately sized dataset, simple methods such as Naive Bayes classification tend to outperform deep learning as the noise in the dataset increases [@doi:10.1186/s13321-017-0226-y].
```

Comment:
I haven't read the cited article, but is this supposed to suggest that the `increased noise` in the second case supposed to contrast the `moderate size` of the dataset that DL performs well on? The two aren't mutually exclusive, but the current wording suggests that they are.

Reply:
The main point I had in mind was that given a moderately sized dataset (let's assume fixed size), if the dataset is noisy, naive Bayes performed better than DL. Probably not the case as the dataset grows, but that wasn't investigated in the study. I think that rephrasing it to 

> Another example is provided by Koutsoukas et al., who benchmarked several traditional machine learning approaches against deep neural networks for modeling bioactivity data on moderately sized datasets. The researchers found that while well tuned deep learning approaches generally tend to outperform conventional classifiers, simple methods such as Naive Bayes classification tend to outperform deep learning as the noise in the dataset increases [@doi:10.1186/s13321-017-0226-y].

what do you think?

Reply:
Yeah, this change would be perfect. You will need to split into two lines and probably want to move the citation to the first sentence.